.env

```diff
@@ -0,0 +1 @@
+MONGOHQ_URL=mongodb://localhost/kiba-cfh
```

Comment:
Please untrack this file. It contains sensitive data and shouldn't be committed to the repo.